core/tests/exhaustive_sync/experiment.rs

```diff
@@ -35,7 +37,8 @@ impl Experiment {
         let mut state = experiments.lock().unwrap();
         let experiment = state.pending.pop();
         match experiment {
-            Some(found) => {
+            Some(mut found) => {
+                found.time_of_start = get_time().0;
```

Comment:
I think this internal modification of a `Trial` should just happen in trial when in `execute()`.

Reply:
I think the default start time should be `0`

additionally when a mutation is generated there is a fn that will reset it's status and other properties, you should set the time on the mutation back to 0 as well.